_src/_posts/2016-08-01-mobile-fraud.markdown

```diff
@@ -141,10 +135,9 @@ If you found this post interesting and want to do stuff like this on a daily bas
  [Kochava]:  https://www.kochava.com/kochava-difference/fraud-detection/
 [unsupervised]: https://en.wikipedia.org/wiki/Unsupervised_learning
 
-<a name="generalTraffic">1</a>:  There are legitimate cases of traffic coming from these sources such as from large organizations, universities and such; however this is not the general case.
-
-<a name="actionFarm">2</a>:  We've evolved the term click farm into action farm to better suit the mobile world since having people manually click, install and use an app is also carried out as fraud. Remember that payback on installs is significantly higher than on clicks or views.
-
-<a name="installDefinition">3</a>: Here an install is defined as the user's first in-app event reported by the tracking platform.
+##References
 
-<a name="anomaly">4</a>: For a more accurate and complete overview of anomaly detection, a very useful survey on the topic can be found at [http://cucis.ece.northwestern.edu/projects/DMS/publications/AnomalyDetection.pdf](http://cucis.ece.northwestern.edu/projects/DMS/publications/AnomalyDetection.pdf).
+[^1]: There are legitimate cases of traffic coming from these sources such as from large organizations, universities and such; however this is not the general case.
+[^2]: We've evolved the term click farm into action farm to better suit the mobile world since having people manually click, install and use an app is also carried out as fraud. Remember that payback on installs is significantly higher than on clicks or views.
+[^3]: Here an install is defined as the user's first in-app event reported by the tracking platform.
```

Comment:
@tzulberti-jampp there is not ^3 but I need @jdemonasterio to confirm where is this coming from since on the original post the reference was "broken" also